projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -565,9 +570,14 @@ export class ExplorerDashboardBuilder {
     }
   }
 
-  private getUserRequestedNonDefaultColumns(
+  private removeColumnsDuplicated(columns: ModelJson[]): ModelJson[] {
+    return uniqBy(columns, 'title'); // Todo: this should be improved
```

Comment:
can we do this by attribute rather than title?

Reply:
updated.